REVEL.pm

```diff
@@ -77,6 +78,41 @@ sub new {
 
   $self->get_user_params();
 
+ # get column count in REVEL file from header line
+  my $file = $self->params->[0];
+  $self->add_file($file);
+  open HEAD, "tabix -fh $file 1:1-1 2>&1 | ";
+  while(<HEAD>) {
+    next unless /^\#/;
+    chomp;
+    $_ =~ s/^\#//;
+    $self->{headers} = [split];
+  }
+  close HEAD;
+
+  die "ERROR: Could not read headers from $file\n" unless defined($self->{headers}) && scalar @{$self->{headers}};
+  my $column_count = scalar @{$self->{headers}};
+  if ($column_count != 7 && $column_count != 8) {
+    die "ERROR: Column count must be 8 for REVEL files with GRCh38 positions or 7 for REVEL files with GRCh37 positions only.\n";
+  }
+  $self->{revel_file_columns} = $column_count;
+
+  my $assembly = $self->{config}->{assembly};
+
+  foreach my $assembly_to_pos (['GRCh37', 'hg19_pos'], ['GRCh38', 'grch38_pos']) {
```

Comment:
Suggest not using a foreach loop as only have one assembly to check. Suggest have a hash of assembly to header e.g.
```
my %assembly_to_hdr = ('GRCh37' => 'hg19_pos',
                         'GRCh38' => 'grch38_pos');

  if (! grep {$_ eq $assembly_to_hdr{$assembly}} @{$self->{headers}}) {
      die "ERROR: Assembly is " . $assembly .
          " but REVEL file does not contain " .
          $assembly_to_hdr{$assembly} . " in header.\n";
  }
```

Reply:
That is better! I will update the PR.